tests/api_tests/nodes/test_views.py

```diff
@@ -848,6 +848,9 @@ def setUp(self):
 
         self.user_two = AuthUserFactory()
 
+        self.deleted_project = NodeFactory(parent=self.public_project, is_deleted=True, creator=self.user)
```

Comment:
Actually, just in case, add a child node in your test, make sure you can see it, then delete it using a standard deletion method. That way, if we stop using is_deleted, the test will warn us if the API isn't handling it properly.